src/main/scala/it/pagopa/interop/agreementprocess/common/system/ApplicationConfiguration.scala

```diff
@@ -16,13 +16,12 @@ object ApplicationConfiguration {
   val attributeRegistryManagementURL: String =
     config.getString("agreement-process.services.attribute-registry-management")
 
-  val consumerDocumentsPath: String = config.getString("agreement-process.storage.consumer-documents-path")
-
   require(jwtAudience.nonEmpty, "Audience cannot be empty")
 
   val storageKind: String           = config.getString("agreement-process.storage.kind")
   val storageContainer: String      = config.getString("agreement-process.storage.container")
   val agreementDocPath: String      = config.getString("agreement-process.storage.agreement-contract-path")
+  val consumerDocumentsPath: String = config.getString("agreement-process.storage.consumer-documents-path")
```

Comment:
these paths should not be required if this service does not put documents, right?

Reply:
That value is used in `deleteAgreement`.

Reply:
not entirely sure, but shouldn't the document path already contain the full path for the delete?
for example, for risk analysis I see this path
```
dev/risk-analysis/docs/ab83fa99-9fdf-4964-85a1-a21781d037b5/20220927064643_cb399e4f-6311-488d-8036-3210fc4a84df_risk_analysis.8345577850513068469.pdf
```